notebooks/HASP/FluxScaleTutorial/FluxScaleTutorial.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>I am leaving the same feedback I left for the coadd notebook: When attempting to activate the&nbsp;<code>hasp-env</code>&nbsp;conda environment with&nbsp;<code>!conda activate</code>&nbsp;directly in a Jupyter cell, it may lead to a 'Command Not Found' error. To address this, I recommend incorporating conda environment activation as part of the setup instructions for the notebook. Alternatively, consider adding a step within the notebook to install the required dependencies using&nbsp;<code>!pip install -r requirements.txt</code>&nbsp;from a code cell, ensuring a smoother setup process.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/172/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>